scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -172,33 +173,13 @@
     </td>
     <td>
       {% if case.clinvar_variants %}
-        <a href="#clinvar_{{case.display_name}}" data-bs-toggle="modal">{{case.clinvar_variants|length}} {% if case.clinvar_variants|length >1 %} variants {% else %} variant {% endif %}</a>
-        <!-- Modal -->
-        <div class="modal fade" id="clinvar_{{case.display_name}}" tabindex="-1" role="dialog" aria-labelledby="variantModalLabel" aria-hidden="true">
-          <div class="modal-dialog" role="document">
-            <div class="modal-content">
-              <div class="modal-header">
-                <h5 class="modal-title" id="variantModalLabel">Variants included in Clinvar submissions:</h5>
-              </div>
-              <div class="modal-body">
-                <ul>
-                  {% for varname, clinvar in case.clinvar_variants.items() %}
-                    <li>
-                    {% if clinvar.category == 'snv' %}
-                      <a href="{{ url_for('variant.variant', institute_id=case.owner, case_name=case.display_name, variant_id=clinvar.local_id) }}" target="_blank">chr{{clinvar.chromosome}}:{{clinvar.start}}_{{clinvar.ref}}>{{clinvar.alt}}</a>
-                    {% else %}
-                      <a href="{{ url_for('variant.sv_variant', institute_id=case.owner, case_name=case.display_name, variant_id=clinvar.local_id) }}" target="_blank">chr{{clinvar.chromosome}}:{{clinvar.breakpoint1}}_{{clinvar.var_type}}</a>
-                    {% endif %}
-                    </li>
-                  {% endfor %}
-                </ul>
-              </div>
-              <div class="modal-footer">
-                <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-              </div>
-            </div>
-          </div>
-        </div>
+        <span class="badge rounded-pill bg-dark" data-bs-toggle="tooltip" title="ClinVar"><em class="fas fa-database"></em></span>
```

Comment:
👍

Reply:
Perhaps also note in the tooltip where more info about the different submissions can be found - like on the case page, or on the institute ClinVar submissions page?

Reply:
Good suggestion! Done!